pyppeteer/accessibility.py

```diff
@@ -0,0 +1,269 @@
+from typing import List, Dict, Union, Set
+
+from pyppeteer.element_handle import ElementHandle
+
+
+class Accessibility(object):
+    def __init__(self, client):
+        self._client = client
+
+    async def snapshot(self, interstingOnly: bool = None, root: ElementHandle = None):
+        nodes = await self._client.send('Accessibility.getFullAXTree')
+        backendNodeId = None
+        if root:
+            node = await self._client.send(
+                'DOM.describeNode', {
+                    'objectId': 'root._remoteObject.objectId',
+                }
+            )
+            backendNodeId = node['backendNodeId']
+        defaultRoot = AXNode.createTree(nodes)
+        needle = defaultRoot
+        if backendNodeId:
+            needle = [
+                node for node in defaultRoot.find(
+                    lambda node: node._payload.backendDOMNodeId == backendNodeId)
+            ]
+            if not needle:
+                return
+            needle = needle[0]
+        if not interstingOnly:
+            return serializeTree(needle)[0]
+
+        interestingNodes = set()
+        collectInterstingNodes(interestingNodes, defaultRoot, False)
+        if needle not in interestingNodes:
+            return None
+        return serializeTree(needle, interestingNodes)[0]
+
+
+class AXNode(object):
+    def __init__(self, payload: Dict):
+        """
+        :param payload: dict with keys nodeId, name, role, properties
+        """
+        self._payload = payload
+        self._children: List['AXNode'] = []
```

Comment:
No need for a forward reference, already defined at this point
```suggestion
        self._children: List[AXNode] = []
```